resources/lang/de_DE/default.po

```diff
@@ -2173,7 +2176,7 @@ msgstr "Hilfe benötigt"
 
 #: includes/view/ShiftCalendarRenderer.php:314
 msgid "Other angeltype needed / collides with my shifts"
-msgstr "Andere Engeltypen benötigt / kollidiert mit meinen Schichten"
+msgstr "Anderer Engeltyp erforderlich / kollidiert mit meinen Schichten"
```

Comment:
Nicht benutzer?

Reply:
What do you mean? In this case I changed `benötigt` to `erforderlich` because `benötigt` kind of implies that somebody is needed. But it is not a bad thing if workplace shifts stay unbooked.

Reply:
Also see https://github.com/stuvusIT/handtuchsystem/pull/2/files#diff-3dda712beb445dc926265a69e01c2248R208

Reply:
Also see https://github.com/stuvusIT/handtuchsystem/pull/2/commits/ab920c275789cc8339252b6ddd797f73b7571a39#diff-3dda712beb445dc926265a69e01c2248R208